calico-cloud_versioned_docs/version-3.15/get-started/connect/cc-arch-diagram.mdx

```diff
@@ -0,0 +1,51 @@
+|                                | **compliance-reporter**<br />Handles report generation. Reads configuration history from Elasticsearch and determines time evolution of cluster-wide configuration, including relationships between policies, endpoints, services, and Network sets. Data is then passed through a zero-trust aggregator to determine the “worst-case outliers” in the reporting interval.compliance-controllerReads report configuration, and manages creation, deletion, and monitoring of report generation jobs. |                                                              |
+|                                | **compliance-server** <br />Provides the API for listing, downloading, and rendering reports, and RBAC by performing authentication and authorization through the Kubernetes API server. RBAC is determined from the users RBAC for the Global report type and Global report resources. |                                                              |
+|                                | **compliance-benchmarker**<br />A daemonset that runs checks in the CIS Kubernetes Benchmark on each node so you can see if Kubernetes is securely deployed. |                                                              |
+| Image Assurance                | Identifies vulnerabilities in container images that you deploy to Kubernetes clusters.  Includes these components: <br /><br />**Admission controller**<br />Uses Kubernetes Validating Webhook Configuration https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/ to control which images can be used to create pods based on scan results.  <br /><br />**CLI scanner**<br />A CLI binary tool that scans images. Users can upload the results of a scanned image to the backend IA API.  <br /><br />**Pod watcher** <br />Monitors clusters and collects runtime information about which pods are running which images. <br /> | &bull; TCP 8080 to Guardian<br /> &bull;  TCP 6443 to Kubernetes API server |
+|                                | **API and API proxy**<br />Isolates tenant data and ensures that all external access to Image Assurance data is through the API Proxy, which enforces authorization through Kubernetes RBAC. <br /><br />**Note** {{prodname}} does not store registry credentials in its database and does not pull customer images into the {{prodname}} control plane. {{prodname}} uses Google’s encryption at rest feature to encrypt data across the entire datastore. |                                                              |
```

Comment:
@bmckercher123, based a point that @tmjd made about focusing the documentation on what is in the customer's managed cluster only, you should reconsider the inclusion of the following components: 

- Pod watcher is going away in the next release (but even if it wasn't, it's not part of the managed cluster). 

- API and API proxy are not in the managed cluster. We definitely shouldn't mention API proxy, if we don't want to expose any details about the internals of the management cluster. We probably want to mention the API, but only at a high-level (no details about where it runs). 

- CLI scanner might be something we want to mention, but it also does not run in the managed cluster. Or more specifically, the user gets to decided where to run it, but it is not something that is assumed to be running in the managed cluster.

Also, the top of this Image Assurance section says: 

> Identifies vulnerabilities in container images that you deploy to Kubernetes clusters. **Includes these components:**

With that wording (the part in **bold**), it feels like its saying that is a complete list of the components. This might be minor, but I wonder if it would be clearer to say something like "The components of interest are ...".

Reply:
Addressed all comments: deleted pod watcher, left API, deleted CLI scanner. Used "components of interest".